app/merginapi.cpp

```diff
@@ -12,17 +12,18 @@
 MerginApi::MerginApi( const QString &dataDir, QObject *parent )
   : QObject( parent )
   , mDataDir( dataDir )
-  , mCacheFile( QStringLiteral( ".projectsCache.txt" ) )
+  , mCacheFile( QStringLiteral( "/.mergin/mergin.json" ) )
```

Comment:
Couldn't this be a static const string? That shouldn't change in different MerginApi instances...